ex2_hal/iris/equipment_handler/source/iris_spi.c

```diff
@@ -138,9 +122,10 @@ void iris_spi_get(uint16_t *rx_data, uint16_t data_length) {
  *   Number of ticks for non FreeRTOS delay
  *
  **/
-void iris_spi_delay(uint16_t ticks) {
+void iris_spi_delay(uint32_t ticks) {
     uint16_t i;
```

Comment:
uint16 is less than uint32. This has the potential to be an infinite loop

Reply:
Yes, that should I have been fixed now